testnet/etcd/register_etcd.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/sh
+
+END=$DOCKER_LIBP2P_REPLICAS
+NODE_ID=1
+NODE_IP=$(hostname -i)
+NODE_KEY=$(printf '%064s' $(printf '%x' $NODE_ID) | tr ' ' '0')
```

Comment:
```suggestion
NODE_KEY=$(printf '%064s' $(printf '%x' "${NODE_ID}") | tr ' ' '0')
```
This applies to all variables used.